cpp/harvest/marc_issn_lookup.cc

```diff
@@ -395,6 +396,22 @@ void CleanDuplicationOfField773(MARC::Record * const record) {
     }
 }
 
+void CleanDuplicationOfField773ByTitle(MARC::Record * const record) {
+    std::vector<std::string> titles;
+    const ControlNumberGuesser control_number_guesser;
```

Comment:
ControlNumberGuesser::NormaliseTitle is a static function, so it should be possible to call it directly without initializing an instance of the class first.